webtau-http-groovy/src/test/groovy/com/twosigma/webtau/http/HttpGroovyTest.groovy

```diff
@@ -183,6 +191,36 @@ class HttpGroovyTest implements HttpConfiguration {
         assert a == 1
     }
 
+    @Test
+    void "build query params using map helper"() {
+        http.get("params", http.query([a: 1, b: 'text'])) {
+            a.should == 1
+            b.should == 'text'
+        }
+    }
+
+    @Test
+    void "build query params using var arg helper"() {
+        http.get("params", http.query('a', '1', 'b', 'text')) {
+            a.should == 1
```

Comment:
this example may be confusing for docs usage, as it is not clear why `a` should equal 1. 
Headers documentation just use `// assertion goes here`. That means that you may need another test to just validate syntax, but have no actual validation.

Reply:
Updated